### PR TITLE
🍰 2019/kw21/github_templates_update

### PR DESCRIPTION
**Pull request by Tirokk**
> [ulfgebhardt](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-20T09:25:37Z" title="Monday, May 20th 2019, 11:25:37 am +02:00">May 20, 2019</time>_
_Merged <time datetime="2019-05-21T13:25:40Z" title="Tuesday, May 21st 2019, 3:25:40 pm +02:00">May 21, 2019</time>_
---

## 🍰 Pullrequest

- more emoji
- removed unused sections
- more meta data
- assign new issues to ulfgebhardt

### Issues

- None

### Todo

- [ ] Does it work when merged?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 name: 💬 Question
 about: If you need help understanding HumanConnection.
+labels: question
+title: 💬 [Question]
```
Comment (Tirokk) [inline .github/ISSUE_TEMPLATE/question.md:5]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-20T10:55:47Z" title="Monday, May 20th 2019, 12:55:47 pm +02:00">May 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,6 +1,9 @@
 ---
 name: 💬 Question
 about: If you need help understanding HumanConnection.
+labels: question
+assignees: ulfgebhardt
```


```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,27 +1,12 @@
-## Pullrequest
-<!-- Describe the Pullrequest. -->
+## 🍰 Pullrequest
+<!-- Describe the Pullrequest. Use Screenshots if possible. -->
```
Comment (Tirokk) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-20T10:56:31Z" title="Monday, May 20th 2019, 12:56:31 pm +02:00">May 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,27 +1,12 @@
-## Pullrequest
-<!-- Describe the Pullrequest. -->
+## 🍰 Pullrequest
+<!-- Describe the Pullrequest. Use Screenshots if possible. -->
```


:+1:

**Comment by Tirokk** [comment]
> [codecov[bot]](https://github.com/apps/codecov) **Authored by [codecov[bot]](https://github.com/apps/codecov)**
_<time datetime="2019-05-21T12:20:55Z" title="Tuesday, May 21st 2019, 2:20:55 pm +02:00">May 21, 2019</time>_

---


# [Codecov](https://codecov.io/gh/Human-Connection/Human-Connection/pull/633?src=pr&el=h1) Report
> Merging [#633](https://codecov.io/gh/Human-Connection/Human-Connection/pull/633?src=pr&el=desc) into [master](https://codecov.io/gh/Human-Connection/Human-Connection/commit/859d3189f4b2c8f3df255d60042d47bcf6186e3a?src=pr&el=desc) will **not change** coverage.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/Human-Connection/Human-Connection/pull/633?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master     #633   +/-   ##
=======================================
  Coverage   20.11%   20.11%           
=======================================
  Files         157      157           
  Lines        2208     2208           
  Branches      105      105           
=======================================
  Hits          444      444           
  Misses       1740     1740           
  Partials       24       24
```

**Review by Mogge — APPROVED** [review]
:clap:

**Pull request closed without merging**